[2023-11-23T20:33:19.332+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Projet1.Extract manual__2023-11-23T20:33:02.649387+00:00 [queued]>
[2023-11-23T20:33:19.343+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Projet1.Extract manual__2023-11-23T20:33:02.649387+00:00 [queued]>
[2023-11-23T20:33:19.343+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-23T20:33:19.356+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Extract> on 2023-11-23 20:33:02.649387+00:00
[2023-11-23T20:33:19.362+0000] {standard_task_runner.py:57} INFO - Started process 2151 to run task
[2023-11-23T20:33:19.366+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'Projet1', 'Extract', 'manual__2023-11-23T20:33:02.649387+00:00', '--job-id', '713', '--raw', '--subdir', 'DAGS_FOLDER/Projet.py', '--cfg-path', '/tmp/tmpp2_o2fyc']
[2023-11-23T20:33:19.367+0000] {standard_task_runner.py:85} INFO - Job 713: Subtask Extract
[2023-11-23T20:33:19.418+0000] {task_command.py:416} INFO - Running <TaskInstance: Projet1.Extract manual__2023-11-23T20:33:02.649387+00:00 [running]> on host b36ebcee5948
[2023-11-23T20:33:19.504+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Projet1' AIRFLOW_CTX_TASK_ID='Extract' AIRFLOW_CTX_EXECUTION_DATE='2023-11-23T20:33:02.649387+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-23T20:33:02.649387+00:00'
[2023-11-23T20:33:19.740+0000] {warnings.py:109} WARNING - /opt/***/dags/Projet.py:34: DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.
  df_urgences = pd.read_csv(csv_file_path_urgences, delimiter=';')

[2023-11-23T20:33:20.126+0000] {logging_mixin.py:151} INFO - Index(['code_age', 'agess'], dtype='object')
[2023-11-23T20:33:20.127+0000] {logging_mixin.py:151} INFO - Une erreur s'est produite lors de l'extraction des donn√©es Age : 'DataFrame' object has no attribute 'append'
[2023-11-23T20:33:20.151+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T20:33:20.233+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T20:33:20.280+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T20:33:41.658+0000] {logging_mixin.py:151} INFO - ########################################################################
[2023-11-23T20:33:41.659+0000] {logging_mixin.py:151} INFO - #                                                                      #
[2023-11-23T20:33:41.659+0000] {logging_mixin.py:151} INFO - #   ce dessous est la visualisation du contenu de la table Urgences :  #
[2023-11-23T20:33:41.659+0000] {logging_mixin.py:151} INFO - #                                                                      #
[2023-11-23T20:33:41.660+0000] {logging_mixin.py:151} INFO - ########################################################################
[2023-11-23T20:33:41.666+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-23T20:33:41.705+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:219: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2023-11-23T20:33:41.763+0000] {logging_mixin.py:151} INFO -         h      f  dep
0   382.0  427.0    0
1   277.0  346.0    1
2   494.0  524.0    2
3   250.0  269.0    3
4   175.0  205.0    4
..    ...    ...  ...
95  293.0  284.0  971
96  218.0  225.0  972
97  121.0  174.0  973
98  453.0  512.0  974
99    6.0    6.0  976

[100 rows x 3 columns]
[2023-11-23T20:33:41.764+0000] {logging_mixin.py:151} INFO - ========================
[2023-11-23T20:33:42.338+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-23T20:33:42.349+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Projet1, task_id=Extract, execution_date=20231123T203302, start_date=20231123T203319, end_date=20231123T203342
[2023-11-23T20:33:42.380+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-23T20:33:42.401+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
